### PR TITLE
Fix Friendlist disappear in 2020-09-02ragexeRE or above

### PR DESCRIPTION
**Pull request by attackjom**
- fix friend list disappear in client 2020-09-02ragexeRE+

* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/5582

* **Server Mode**: Renewal

* **Description of Pull Request**: 
Fix a problem that friend list disappear in 2020-09-02ragexeRE.

### CODE REVIEW

**Review by Lemongrass3110 — CHANGES_REQUESTED** [review]
Your check has to be changed. If PACKETVER_RE_NUM is not defined it will result in 0 < 20200902 and will therefore be true.

File: src/map/clif.cpp [hunk 1]
```diff
@@ -14873,7 +14873,7 @@ int clif_friendslist_toggle_sub(struct map_session_data *sd,va_list ap)
 void clif_friendslist_send(struct map_session_data *sd)
 {
 	int i = 0, n, fd = sd->fd;
-#if PACKETVER >= 20180221
+#if PACKETVER >= 20180221 && !(PACKETVER_RE_NUM >= 20200902)
 	const int size = 8;
 #else
 	const int size = 8 + NAME_LENGTH;
```
Comment (Lemongrass3110) [inline src/map/clif.cpp:14879]:
```suggestion
#if PACKETVER < 20180221 || PACKETVER_RE_NUM >= 20200902
	const int size = 8 + NAME_LENGTH;
#else
	const int size = 8;
```

**Review by Lemongrass3110 — APPROVED** [review]
Just a "beauty" correction, to match the other ifdef.

**Comment by Balferian** [comment]
After this fix nothing changed.
Tested on **2020-09-02aRagexeRE** and **2020-09-02bRagexeRE**.
(UI) After relogin friend list is clear.
image

**Comment by attackjom** [comment]
> After this fix nothing changed.
> Tested on **2020-09-02aRagexeRE** and **2020-09-02bRagexeRE**.
> (UI) After relogin friend list is clear.
> image

you need to defind

#define PACKETVER_RE_NUM 20200902  in packet.hpp or define_post.hpp

**Comment by Balferian** [comment]
> > After this fix nothing changed.
> > Tested on **2020-09-02aRagexeRE** and **2020-09-02bRagexeRE**.
> > (UI) After relogin friend list is clear.
> 
> you need to define
> 
> #define PACKETVER_RE_NUM 20200902 in packet.hpp or define_post.hpp
@attackjom Thanks! Problem solved.

**Pull request merged**